sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/spark.py

```diff
@@ -474,8 +478,54 @@ def _customized_validate(self):
                 yaml_path="environment.image",
                 message=SPARK_ENVIRONMENT_WARNING_MESSAGE,
             )
+        result.merge_with(self._validate_entry_exist(raise_error=False))
         return result
 
+    def _validate_entry_exist(self, raise_error=False) -> MutableValidationResult:
```

Comment:
May remove this check since there will be validation error in pipeline submission time

Reply:
Can add an e2e test to check the validation error in submission time

Reply:
you mean we will also validate entry path also exists? this function validate entry is not NOne and entry path exists

Reply:
@D-W- , I think we need this func to check entry path exists, because it seems submittion won't validate it

Reply:
remove validate whether entry is None here, leave it to validate in submit